src/Hooks/useModal.jsx

```diff
@@ -0,0 +1,12 @@
+import { useState } from "react";
```

Comment:
This component doesn't seem to be used.

Reply:
This will end up being used once components are laid out.

Reply:
This component was created and used in FRG3D-5.
Now that that is merged into the main branch there's no issue but keep in mind of best practices
We always want a single source of truth instead of two spidermans pointing at each other cause they catching bugs